src/Resources/config/tool.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="stof_doctrine_extensions.tool.actor_provider.class">Stof\DoctrineExtensionsBundle\Tool\TokenStorageActorProvider</parameter>
+    </parameters>
+    <services>
+        <service id="stof_doctrine_extensions.tool.actor_provider" class="%stof_doctrine_extensions.tool.actor_provider.class%">
```

Comment:
```suggestion
        <service id="stof_doctrine_extensions.tool.actor_provider" class="%stof_doctrine_extensions.tool.actor_provider.class%" public="false">
```

The service should be private. We don't need to make it available through `$container->get()` at runtime as the expected usage is only to inject it in other services.

Reply:
Updated